Sequencer/Sequencer/TcSequencer/Example/Mock_EM.TcPOU

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <POU Name="Mock_EM" Id="{2fbe5d7f-06dd-041a-2d03-5fb4cef27abb}" SpecialFunc="None">
+    <Declaration><![CDATA[
+FUNCTION_BLOCK Mock_EM IMPLEMENTS IMOCK_EM
+VAR_INPUT
+  TimeUp : TIME;
+  TimeDown : TIME;
+END_VAR
+VAR
+  _MoveUp : PLCopenCommand;
+  _MoveDown : PLCopenCommand;
+  _HeightGroup : PLCopenExclusiveGroup := (commands := [_MoveUp, _MoveDown, 18(0)]); 
+  t : TON;
+  Up : BOOL;
+  Down : BOOL;
+  Up_ : BOOL;
```

Comment:
What's the distinction between Up_ and Up? This could be confusing to new users in the example without a comment.